### PR TITLE
Add Longhorn 1.5.3 chart with dev-2.7

### PR DESCRIPTION
**Pull request by mantissahz**
release longhorn v1.5.3 into Rancher 2.7

Longhorn longhorn/longhorn#7082
Rancher rancher/rancher#43566

Signed-off-by: James Lu <james.lu@suse.com>

cc @innobead @PhanLe1010 @khushboo-rancher @roger-ryao

### CODE REVIEW

**Comment by mantissahz** [comment]
Self-Verified: PASSED
Install:
Screenshot_20231120_225127

Workload & Recurring Job Backup:
2023-11-20pm10-55-11

Uninstall:
Screenshot_20231120_225713

**Comment by roger-ryao** [comment]
Prereq: Set Concurrent Automatic Engine Upgrade Per Node Limit to 0
Test steps:
- [x] 1. Fresh installation
- [x] 2. Uninstallation
- [x] 3. Upgrade from v1.4.4
- [x] 4. Upgrade from v1.5.1

**Review by PhanLe1010 — APPROVED** [review]
~~LGTM~~

File: charts/longhorn/102.3.1+up1.5.3/questions.yaml.orig [hunk 1]
```diff
@@ -0,0 +1,890 @@
+categories:
```
Comment (PhanLe1010) [inline charts/longhorn/102.3.1+up1.5.3/questions.yaml.orig]: This file is unnecessary. Can we make sure to cleanup and rerun the chart making follow https://confluence.suse.com/display/LON/How+to+create+Rancher+Chart+PRs

Comment (mantissahz) [reply charts/longhorn/102.3.1+up1.5.3/questions.yaml.orig]: Removed files and updated the confluence.
Thanks

File: charts/neuvector-crd/102.0.6+up2.6.6/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+annotations:
+  catalog.cattle.io/certified: rancher
+  catalog.cattle.io/hidden: "true"
+  catalog.cattle.io/namespace: cattle-neuvector-system
+  catalog.cattle.io/release-name: neuvector-crd
```
Comment (PhanLe1010) [inline charts/neuvector-crd/102.0.6+up2.6.6/Chart.yaml]: Why are we adding neuvector chart here?

Comment (mantissahz) [reply charts/neuvector-crd/102.0.6+up2.6.6/Chart.yaml]: Removed. Restarted the process from scratch.
Thanks.

**Comment by mantissahz** [comment]
@roger-ryao 
Could you re-test the new pushed files again when you are available? Thanks.

**Review by PhanLe1010 — APPROVED** [review]

**Comment by roger-ryao** [comment]
> @roger-ryao Could you re-test the new pushed files again when you are available? Thanks.

- [x] Fresh installation
- [x] Uninstallation
- [x] Upgrade from v1.5.1 → v1.5.3
- [x] Upgrade from v1.4.4 → v1.5.3

**Pull request merged**